bin/porta

```diff
@@ -432,31 +432,31 @@ module Porta
 
     protected
 
-    def docker_internal_host
-      options[:docker_internal_host] ||= container_local_host_ip
+    def docker?
+      docker = system('docker --version > /dev/null 2>&1')
+    end
+
+    def container_tool
+      docker? ? 'docker' : 'podman'
+    end
```

Comment:
I would give priority to `podman`, since it's the officially supported tool in RHEL. Also, adding simple memoization through an instance variable could save many calls to `system`.

```suggestion
    def container_tool
      @container_tool ||= podman? ? 'podman' : 'docker'
    end
```